modules/hydropower/workflow/internal/internal_config.yaml

```diff
@@ -0,0 +1,49 @@
+# This file contains configuration that users should not modify.
+resources:
+  default_user_resources:
```

Comment:
Slight preference for not nesting these and providing resources and default_user_resources on the same level.

Reply:
I've kept the `resources` nesting (I think it's just a bit tidier), but moved the default user resources to the same level.